opentelemetry/instrumentation/app/main_test.go

```diff
@@ -123,7 +123,7 @@ func TestWriteTelemetry(t *testing.T) {
 		select {
 		case <-time.After(timeoutSeconds * time.Second):
 			t.Error("Timeout waiting for metrics")
-		case <-ts.expectationsMet:
+		case <-ms.expectationsMet:
```

Comment:
this fixes a bug where we were checking the trace expectations in the metric test, and thus were not testing metrics.  Notice the typo in the metric name on line 71.